ci/bootstrap.sh

```diff
@@ -1,4 +1,15 @@
 #!/usr/bin/env bash
```

Comment:
Make sure to `chmod +x` on all of these shell scripts

Reply:
yeah, please do that